Sources/Logging/Strings/StoreKitStrings.swift

```diff
@@ -59,6 +59,16 @@ enum StoreKitStrings {
 
     case sk2_observing_transaction_updates
 
+    case sk2_unknown_environment(String)
+
+    case sk2_error_encoding_receipt(Error)
+
+    case sk2_error_fetching_app_transaction(Error)
+
+    case sk2_app_transaction_unavailable
+
+    case sk2_unverified_transaction(String, Error)
```

Comment:
Let's put a label to `String` since it's not obvious what it is based on the type alone:
```suggestion
    case sk2_unverified_transaction(identifier: String, Error)
```